docs/howtos/telemetry/metrics/_category_.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I wonder if for opentelemetry, tracing, and metrics we need collapsed subsections that are empty. I would be ok showing in the sidebar the following:
> - Telemetry
>   * Opentelemetry quickstart
>   * Tracing quickstart
>   * Metrics quickstart


I don't expect each of the OTLM, Tracing, Metrics to grow; new production deployments will likely be downstream in the UI and automated, and will take a while to get community submissions to add to the docs.

Reply:
Ok, I've done that for the next commit.